### ISSUE TITLE
a11y: Table header and cell roles

### ISSUE THREAD

**Issue by jchen527** [bvaughn/react-virtualized#680]
The table implementation is missing a few other ARIA roles:

- Table headers should have `role="columnheader"`, an `id` to associate the column's cells, and `aria-sort` if the column is sorted
- Table cells should have `role="gridcell"` and `aria-describedby` that points to the associated column header

ARIA documentation: [columnheader](https://www.w3.org/TR/wai-aria/roles#columnheader) and [gridcell](https://www.w3.org/TR/wai-aria/roles#gridcell).
